carma_novatel_driver_wrapper/launch/carma-novatel-driver-wrapper-launch.py

```diff
@@ -0,0 +1,59 @@
+
```

Comment:
There is a mis-commited file above this
carma_novatel_driver_wrapper/launch/__pycache__/carma_novatel_driver_launch.cpython-38.pyc